src/main/kotlin/no/nav/syfo/aksessering/api/BehandlerApi.kt

```diff
@@ -19,11 +19,11 @@ fun Route.registerBehandlerApi(elektroniskAbonomentService: ElektroniskAbonoment
             if (herid.isNullOrEmpty()) {
                 log.info("Mangler query parameters: herid")
                 call.respond(HttpStatusCode.BadRequest)
-            } else if (elektroniskAbonomentService.finnParnterInformasjon(herid).isEmpty()) {
-                log.info("Fant ingen elektroniskAbonoment for akutell herid")
-                call.respond(emptyList<ElektroniskAbonoment>())
+            } else if (partnerInformasjonService.finnPartnerInformasjon(herid).isEmpty()) {
+                log.info("Fant ingen partnerInformasjon for akutell herid")
+                call.respond(emptyList<PartnerInformasjon>())
             } else {
-                call.respond(elektroniskAbonomentService.finnParnterInformasjon(herid))
+                call.respond(partnerInformasjonService.finnPartnerInformasjon(herid))
```

Comment:
Uavhengig av denne PRen: Er det ikke litt unødvendig å gjøre `finnPartnerInformasjon`-kallet to ganger?

Reply:
Jo kanskje, er bare å herje i denne appen, så mye derre vil og ta over eierskapet til den

Reply:
Jo, jeg tenkte på det. Men mangel på Kotlin foo stoppet meg fra å gjøre det bedre. Hvordan returnerer jeg fra funksjonen? Kan man gjøre return void?